packages/core/src/app/address/googleAutocomplete/GoogleAutocomplete.tsx

```diff
@@ -45,7 +45,8 @@ class GoogleAutocomplete extends PureComponent<GoogleAutocompleteProps, GoogleAu
 
         return (
             <Autocomplete
-                initialHighlightedIndex={0}
+                defaultHighlightedIndex={-1}
+                initialHighlightedIndex={-1}
```

Comment:
In downshift, disabling automatic highlight takes two options. ([link](https://github.com/downshift-js/downshift#initialhighlightedindex))
- `defaultHighlightedIndex`: This is the value to set the highlight item when downshift is reset, such as the selection is cleared, an item is selected, the `inputValue` is changed.
- `initialHighlightedIndex`: This is the initial index to highlight when the menu first opens.